src/consumption.cpp

```diff
@@ -1093,9 +1093,19 @@ static bool eat( item &food, Character &you, bool force )
 void Character::modify_health( const islot_comestible &comest )
 {
     const int effective_health = comest.healthy;
-    // Effectively no cap on health modifiers from food and meds
-    const int health_cap = 200;
-    mod_daily_health( effective_health, effective_health >= 0 ? health_cap : -health_cap );
+    int daily_food_health += effective_health;
+
+    if( calendar::once_every( 1_days ) ) {
```

Comment:
Lines 1098-1108 should be moved to `update_body` in `character_body.cpp` as `modify_health` is only called when a comestible is consumed.